README.md

```diff
@@ -1,4 +1,15 @@
-# Gotrue-py
+# gotrue-py
+
+[![License: MIT](https://img.shields.io/badge/License-MIT-green.svg?label=license)](https://opensource.org/licenses/MIT)
+[![CI](https://github.com/leynier/gotrue-py/actions/workflows/ci.yml/badge.svg)](https://github.com/leynier/gotrue-py/actions/workflows/ci.yml)
```

Comment:
this is pointing your fork, is this the intended behaviour?

Reply:
Oops, it was to see the result and I forgot to change it 😅

Reply:
no worries, I can change it myself.